ansible/roles/omero-user/README.md

```diff
@@ -0,0 +1,81 @@
+OMERO user
+==========
+
+Create OMERO user accounts and groups.
+This will not modify existing users or groups, apart from the user password if `force` is set.
+
+
+Dependencies
+------------
+
+Assumes the `omero-server` role is installed with defaults, if not you will have to set most of the role variables below.
+
+This requires features present in OMERO 5.X.
+
+
+Role Variables
+--------------
+
+All variables are optional, see `defaults/main.yml`
+
+Create OMERO user accounts and groups:
+- `omero_user_bin_omero`: The full path to `bin/omero` application, default `/home/omero/OMERO.server/bin/omero`
+- `omero_user_system`: Run the `omero` CLI as this user, default `omero` (must not be `root`)
+- `omero_user_admin_user`: Login to OMERO as this admin user, default `root`
+- `omero_user_admin_pass`: Password for `omero_user_admin_user`
+
+- `omero_user_create`: List of dictionaries of OMERO users to create with fields:
+  - `login`: OMERO user-name
+  - `firstname`: First name
+  - `lastname`: Last name
+  - `password`: Password
+  - `groups`: String containing group arguments (see `bin/omero user add --help`)
+  - `force`: Forcibly reset password, default `False` (requires direct database access)
```

Comment:
Yes, you can create non-interactively script the creation of new users with passwords but not reset passwords. I thought about opening an openmicroscopy PR but decided it could benefit from discussion, for example should only admins be allowed to non-interactively reset passwords, like `passwd` vs `chpasswd`